lib/usd/translators/shading/CMakeLists.txt

```diff
@@ -54,10 +62,20 @@ if (MAYA_APP_VERSION VERSION_GREATER 2022)
             mtlxStandardSurfaceWriter.cpp
             mtlxSymmetricShaderReader.cpp
             mtlxSymmetricShaderWriter.cpp
+            mtlxTranslationTableReader.cpp
+            mtlxTranslationTableWriter.cpp
             mtlxFileTextureWriter.cpp
     )
     target_compile_definitions(${TARGET_NAME}
         PRIVATE
         WANT_MATERIALX_TRANSLATORS=1
     )
+    if (USD_HAS_MX_OPENPBR_SURFACE AND MAYA_API_VERSION VERSION_GREATER_EQUAL 20250300)
+        target_sources(${TARGET_NAME}
+            PRIVATE
+                mtlxOpenPBRSurfaceReader.cpp
+                mtlxOpenPBRSurfaceWriter.cpp
```

Comment:
Exporting to MaterialX makes sense only if MaterialX can actually render OpenPBR nodes.